ui/app/src/modules/Search/Search.tsx

```diff
@@ -638,108 +639,113 @@ export default function Search(props: SearchProps) {
+              rowsPerPageOptions={[9, 15, 21]}
+              className={classes.pagination}
+              component="div"
+              labelRowsPerPage={formatMessage(messages.itemsPerPage)}
+              count={searchResults.total}
+              rowsPerPage={searchParameters.limit}
+              page={Math.ceil(searchParameters.offset / searchParameters.limit)}
+              backIconButtonProps={{
+                'aria-label': 'previous page'
+              }}
+              nextIconButtonProps={{
+                'aria-label': 'next page'
+              }}
+              onChangePage={handleChangePage}
+              onChangeRowsPerPage={handleChangeRowsPerPage}
             />
-          </FormGroup>
-          {selected.length > 0 && (
-            <Typography variant="body2" className={classes.resultsSelected} color={'textSecondary'}>
-              {formatMessage(messages.resultsSelected, {
-                count: selected.length,
-                total: searchResults.total
-              })}
-              <HighlightOffIcon className={classes.clearSelected} onClick={handleClearSelected} />
-            </Typography>
+          </div>
+        )}
+        <section className={classes.content}>
+          {apiState.error ? (
+            <ApiResponseErrorState error={apiState.errorResponse} />
+          ) : (
+            <Grid container spacing={3} className={searchResults?.items.length === 0 ? classes.empty : ''}>
+              {searchResults === null ? (
+                <Spinner background="inherit" />
+              ) : (
+                renderMediaCards(searchResults.items, currentView)
+              )}
+            </Grid>
           )}
-          <TablePagination
-            rowsPerPageOptions={[9, 15, 21]}
-            className={classes.pagination}
-            component="div"
-            labelRowsPerPage={formatMessage(messages.itemsPerPage)}
-            count={searchResults.total}
-            rowsPerPage={searchParameters.limit}
-            page={Math.ceil(searchParameters.offset / searchParameters.limit)}
-            backIconButtonProps={{
-              'aria-label': 'previous page'
-            }}
-            nextIconButtonProps={{
-              'aria-label': 'next page'
+        </section>
+        {simpleMenu.item?.path && (
+          <SimpleMenu
+            permissions={permissions?.[simpleMenu.item.path]}
+            item={simpleMenu.item}
+            anchorEl={simpleMenu.anchorEl}
+            onClose={onMenuClose}
+            onNavigate={onNavigate}
+            onEdit={onEdit}
+            onDelete={onDelete}
+            messages={{
+              edit: formatMessage(messages.edit),
+              delete: formatMessage(messages.delete),
+              preview: formatMessage(messages.preview),
+              noPermissions: formatMessage(messages.noPermissions)
             }}
-            onChangePage={handleChangePage}
-            onChangeRowsPerPage={handleChangeRowsPerPage}
           />
```

Comment:
Please replace the simple menu with the ItemMenu (same as PathNavigator)

Reply:
Notice ItemMenu is in the GlobalDialogManager. 
Ask Aaron if you need some guidance to integrate.